### PR TITLE
add text-utilities, lists; add landing page

### PR DESCRIPTION
**Pull request by iRohitGaur**

### CODE REVIEW

File: pages/components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+.sui_li_v3:hover {
+  color: var(--VARIANT-3-TXT-BG);
+}
+.sui_ul_stacked {
+  background: var(--SHADE2);
+  padding: 0;
+  width: 100%;
+}
+li.sui_li_stacked {
```
Comment (zoef-yt) [inline pages/components/lists/lists.css:44]: Won't using the `sui_li_stacked` class work directly?

**Review by zoef-yt — APPROVED** [review]
Looks good to me.

**Review by varun13169 — APPROVED** [review]
Looks good to me

File: pages/components/lists/lists.css (hunk 1 above)
Comment (iRohitGaur) [reply pages/components/lists/lists.css:44]: It wont as the initial class ".sui_ul li" beats this one on specificity.

Comment (zoef-yt) [reply pages/components/lists/lists.css:44]: Got it.

**Pull request merged**